src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -1,9 +1,34 @@
 package core.basesyntax;
 
-/**
- * Feel free to remove this class and create your own.
- */
+import core.basesyntax.service.FruitShopService;
+import core.basesyntax.strategy.BalanceHandler;
+import core.basesyntax.strategy.OperationStrategy;
+import core.basesyntax.strategy.PurchaseHandler;
+import core.basesyntax.strategy.ReturnHandler;
+import core.basesyntax.strategy.SupplyHandler;
+import java.util.HashMap;
+import java.util.Map;
+
 public class HelloWorld {
-    // HINT: In the `public static void main(String[] args)` it is better to create instances of your classes, 
-    // and call their methods, but do not write any business logic in the `main` method!
+    private static final String INPUT_FILE_PATH = "src/main/resources/reportToRead.csv";
+    private static final String OUTPUT_FILE_PATH = "src/main/resources/finalReport.csv";
+
+    public static void main(String[] arg) {
+        Map<String, OperationStrategy> operationStrategies = new HashMap<>();
+        operationStrategies.put("b", new BalanceHandler());
```

Comment:
Create an inner enum OperationType in FruitTransaction class with values for each operation you have:
```
    BALANCE("b"),
    SUPPLY("s"),
    PURCHASE("p"),
    RETURN("r");
```
Same for other handlers
```suggestion
        operationStrategies.put(FruitTransaction.OperationType.BALANCE, new BalanceHandler());
```